src/OrchardCore.Modules/OrchardCore.Media.Azure/OrchardCore.Media.Azure.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.FileStorage.AzureBlob\OrchardCore.FileStorage.AzureBlob.csproj" />
+    <ProjectReference Include="..\OrchardCore.Admin\OrchardCore.Admin.csproj" />
+    <ProjectReference Include="..\OrchardCore.Media\OrchardCore.Media.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="Module.txt">
+      <PackagePath>assets\$(PackageId)\</PackagePath>
+      <Pack>true</Pack>
+    </None>
+    <None Update="README.md">
+      <PackagePath>assets\$(PackageId)\</PackagePath>
+      <Pack>true</Pack>
+    </None>
+  </ItemGroup>
+
```

Comment:
Just for infos, in [this .props file](https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore/OrchardCore.Module.Targets/OrchardCore.Module.Targets.props#L52) we manage all assets files for packaging (excluding those we don't want) ...

So we don't need these kind of `ItemGroup` and i'm used to remove them just to show we don't need them. But, because now VS add them automatically, no sure it is worth but i wanted to give you the infos.